cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -930,6 +930,8 @@ def test_http_update_from_json(self):
     def test_advanced_components_munge_tabs(self):
         """
         Test that adding and removing specific advanced components adds and removes tabs.
+
+        "combinedopenended" has been disabled and so its tabs should not show.
```

Comment:
I haven't been able to find an XBlock which goes into the advanced components list and also has a tab.